packages/components/modal/src/modal.tsx

```diff
@@ -149,6 +150,10 @@ type ModalOptions = Pick<
    * Props to be forwarded to the portal component.
    */
   portalProps?: Omit<PortalProps, "children">
+  /**
+   * Apply styles to ModalContent wrapper elements.
```

Comment:
Change "Apply styles to ModalContent wrapper elements." to "Props for modal container element."

Reply:
I fixed!